### PR TITLE
Graphql setup

### PR DESCRIPTION
**Pull request by jdcarey128**
This PR: 
- Pushes up current GraphQL BE setup for team to build off of 
- Sets up GraphQL for rails 
- Defines Car query and mutation (still needs to have tests)
- Note: we need to add PR template for BE

### CODE REVIEW

File: app/graphql/types/mutation_type.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module Types
+  class MutationType < Types::BaseObject
+
+    field :create_user_car, mutation: Mutations::CreateUserCar
+
+    # TODO: remove me
```
Comment (carson-jardine) [inline app/graphql/types/mutation_type.rb:6]: I think we should remove the example text here

File: app/graphql/types/query_type.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+module Types
+  class QueryType < Types::BaseObject
+    # Add root-level fields here.
+    # They will be entry points for queries on your schema.
+
+    # How to define query types: 
+    # field <:query_name>, resolver: Queries::ClassName(defined in query_file.rb)
+    # resolver: executes the logic in the resolve method of the above class 
+
+    # TODO: remove me
```
Comment (carson-jardine) [inline app/graphql/types/query_type.rb:10]: Sample text removed here as well

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,16 @@
...
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Car.destroy_all
+
+5.times do |i|
+  Car.create!(
+    user_id: i + 1, 
+    make: 'subaru', 
+    model: 'forester', 
+    year: '20010',
```
Comment (carson-jardine) [inline db/seeds.rb:16]: Should we change this to a normal year? 😂

**Review by carson-jardine — APPROVED** [review]
There's some cleanup that needs to be done but overall looks fantastic. Nice work team!!

File: db/seeds.rb (hunk 3 above)
Comment (c-bartell) [reply db/seeds.rb:16]: Do we want/need any sad path testing/responses for if the front end sends blank/invalid data?

**Review by c-bartell — APPROVED** [review]

**Review by ETBassist — APPROVED** [review]
Thanks for putting this all together! Merging this and adding a couple issues as reminders to put together sad path testing

**Pull request merged**